### PR TITLE
Feat/treasury vester

### PR DESCRIPTION
**Pull request by hochbruj**
https://trello.com/c/aDDiN0F7

### CODE REVIEW

File: utils/constants.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const VEX_CONSTANTS = {
     // Just a placeholder, this address does not contain a contract
```
Comment (xenide) [inline utils/constants.js]: Can we remove this comment now that this address actually has a contract?

**Comment by xenide** [comment]
Thanks for the good work, looks pretty good. Some further improvements we can make:

1. As spoken, please review if we really need the additional `BigNumber` library as it seems that the existing `ethers.BigNumber` implementation can handle the operations you're doing. You can refer to other instances of use of `BigNumber` within the same repo. 

2. When the user declines the transaction on sync2, currently it shows up as an unhandled error. We should handle it with a `catch` clause, and just print it to the console or something, but it shouldn't crash like this. 
Screenshot 2022-02-05 at 6 58 36 AM
 
3. The contract address of the TreasuryVester for the DAO should be `0x8Fc3737CF99984961b43f86ce5C82cfaa4B49657`. Please verify again with the output in the governance repo. The current address you have is the vester for proxima. 

Thanks

**Review by xenide — CHANGES_REQUESTED** [review]

File: state/assets.js [hunk 2]
```diff
@@ -75,8 +74,8 @@ function useAssets()
             if (currentBlockTimestamp >= vestingEnd) {
                 claimableAmount = vexBalance;
             } else {
```
Comment (xenide) [inline state/assets.js]: Can we start the else on a newline?

**Comment by xenide** [comment]
Just have that one last comment. Besides that, everything looks good, thanks 👍

**Review by KennethAshley — APPROVED** [review]
Alex commented on the changes I'd make so I'm gonna approve this! Good code!

File: state/assets.js (hunk 2 above)
Comment (hochbruj) [reply state/assets.js]: done

**Pull request merged**